hints/cygwin.sh

```diff
@@ -62,13 +71,23 @@ case "$osvers" in
         d_inetpton='undef'
 esac
 
+case "$osvers" in
+    2.[0-4].*|1.*)
+        # newlib finitel is buggy before cygwin-2.5.0
+        d_finitel='undef'
+        ;;
+esac
+
 # compile Win32CORE "module" as static. try to avoid the space.
 if test -z "$static_ext"; then
   static_ext="Win32CORE"
 else
   static_ext="$static_ext Win32CORE"
 fi
 
+# configury should not check for xlocale.h if it gets the API from locale.h
+i_xlocale='undef'
+
```

Comment:
I think this is obsolete since 68f1b6d70

Reply:
Removed it